src/frame.rs

```diff
@@ -8,15 +9,18 @@ use core::ops::Deref;
 #[cfg(feature = "use_spin")]
 use spin::Mutex;
 
-/// A frame allocator that uses buddy system,
-/// requiring a global allocator
+/// A frame allocator that uses buddy system, requiring a global allocator.
+///
+/// The max order of the allocator is specified via the const generic parameter `ORDER`. The frame
+/// allocator will only be able to allocate ranges of size up to 2\*\*ORDER, out of a total range of
```

Comment:
use `<sup>ORDER</sup>` instead of `**` ?

Reply:
Done, thanks! I didn't know that was possible.